ui/src/components/messages/Messages.tsx

```
@@ -1,59 +1,32 @@
-  },
-}
-
 const Messages = () => {
-  const [messages, setMessages] = useState<Message[]>([])
-  MessageState.set = setMessages
-  MessageState.get = messages
+  const { messages, setMessages } = useMessageState()
 
   const getMessageElement = (message: Message): React.FunctionComponent => {
     const dismissMessage = (message: Message) => {
       message.onDismiss && message.onDismiss()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)